### PR TITLE
Update README to match new Docker server and registry configuration options

### PR DESCRIPTION
**Pull request by carlossg**
@reviewbybees

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,23 @@ Firstly, ensure you have docker running (if you are running with a slave, ensure
...
 You can use the example under [`src/test/example`](https://github.com/jenkinsci/docker-build-publish-plugin/tree/master/src/test/example) to build a very simple busybox based image,
-and push it to `jenkinsci/docker-build-and-publish-example`.
+and push it to `acme/test`.
 
 ![build instructions](https://raw.githubusercontent.com/jenkinsci/docker-build-publish-plugin/master/build-config.png)
 
-The usual Docker build caching mechanism applies - and you can choose to publish, or not, the resultant image, configured under Advanced options.
+A remote Docker server can be configured providing its Docker server URI and creating a `Docker Server Certificate Authentication` credential containing the server certificates.
+
+![server credentials](https://raw.githubusercontent.com/jenkinsci/docker-build-publish-plugin/master/credentials_server.png)
```
Comment (jglick) [inline README.md]: I am guessing simple

```markdown
![server credentials](credentials_server.png)
```

works, no?

File: README.md [hunk 2]
```diff
@@ -37,18 +37,23 @@ Firstly, ensure you have docker running (if you are running with a slave, ensure
...
+
 
-In order to push to a registry, set your credentials (username, email, password) in Manage Jenkins. By default these are used to access the Docker Registry at index.docker.io, this includes private repositories:
+In order to push to a registry, set the `Docker Registry URL` and your credentials (username, password).
```
Comment (jglick) [inline README.md]: Suggest underscores rather than backticks for user labels.

**Comment by jglick** [comment]
:+1:

**Comment by oleg-nenashev** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (carlossg) [inline README.md]: I thought there was a reason for that, copying to the wiki page or rendering it somewhere else, not a big deal

**Pull request merged**